### PR TITLE
release: bump version to 1.5.2

### PR DESCRIPTION
**Pull request by radoering**
### Fixed

- Fix an issue where wheels built on Windows could contain duplicate entries in the RECORD file ([#555](https://github.com/python-poetry/poetry-core/pull/555)).

### CODE REVIEW

**Comment by gpongelli** [comment]
Hi @radoering , do you know when it's planned to merge this PR ? and also, do you know when it'll be then included into poetry? Thanks.

**Comment by radoering** [comment]
It will be merged as soon as another maintainer gives their approval.

As for poetry 1.4.1: I had the hope that we can wait for an [installer](https://github.com/pypa/installer) release but I might prepare it next week anyway.

**Review by neersighted — APPROVED** [review]

**Pull request merged**
